### PR TITLE
Convert to Mdoc

### PR DESCRIPTION
**Pull request by gatorcse**
Convert docs to Mdoc. Tut is out of date and we need to switch off.

### CODE REVIEW

File: site/docs/docs/05-Counter.md [hunk 1]
```diff
@@ -41,19 +41,20 @@ val noLabelsExample = {
       "Example Counter of Failure"
     )
     _ <- IO(println("Action Here")).guaranteeCase{
-      case ExitCase.Completed => successCounter.inc
+      case OutcomeIO.Succeeded(_) => successCounter.inc
```
Comment (gatorcse) [inline site/docs/docs/05-Counter.md:44]: This still isn't compiling. What's the correct way to write this in CE3?

Comment (Kazark) [reply site/docs/docs/05-Counter.md:44]: `OutcomeIO` is a type alias. The correct way to write it is `Outcome.Succeeded(_)`.

**Comment by Kazark** [comment]
I didn't even notice this, but it should now be subsumed in #223

**Comment by gatorcse** [comment]
> I didn't even notice this, but it should now be subsumed in #223
Oh that's _much_ better, looks like you actually knew what you were doing. Closing now.

**Pull request closed without merging**